tests/c_languages/codeclone_detection/ClangASTPrintBearTest.py

```diff
@@ -52,6 +52,6 @@ def test_ast(self):
         ast = "\n"
         # Only check beginning of AST
         for i in range(expected_ast.count("\n")-1):
-            ast += self.queue.get(timeout=0).message + "\n"
+            ast += self.queue.get(timeout=0.1).message + "\n"
```

Comment:
actually there should be no whitespace at all http://pep8.org/#other-recommendations